src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/CheckUtil.java

```diff
@@ -349,4 +351,16 @@ public static String getJavadocTokenText(int[] tokens, int... subtractions) {
         return result.toString();
     }
 
+    public static String getLineSeparatorForFile(String filepath, Charset charset)
+            throws IOException {
+        final String result;
+        if (new FileText(new File(filepath), charset.name())
+                .getFullText().chars().anyMatch(character -> character == '\r')) {
+            result = "\r\n";
```

Comment:
please move this to constant of this Util class, and reuse it FileTextTest.java

Reply:
What exactly do you mean? The `"\r\n"`?

Reply:
If yes, done

Reply:
yes